test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests.csproj

```diff
@@ -30,9 +27,9 @@
     <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.2' ">
     <PackageReference Include="System.Diagnostics.Process" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Net.Http" Version="$(CoreFxVersion)" />
+    <PackageReference Include="System.Net.Http" Version="4.3.2" />
```

Comment:
I have considered having a version for each PackageReference to make sure all assemblies are in sync. It seemed like overkill, because most were the same version, by maybe not.

In this case we would have: 
SystemNetHttpVerson = 4.3.2 (in dependencies and dependenciesTest)

Then in each csproj:
Version="$(SystemNetHttpVersion)"

What do you think?

Reply:
@brentschmaltz - I agree with the suggestion. Changes are included (I forced-pushed the commit).